tldw_Server_API/app/core/Ingestion_Media_Processing/Audio/Audio_Transcription_Lib.py

```
@@ -3192,15 +3244,48 @@ def convert_to_wav(
 
-    try:
-        # Execute ffmpeg command with enhanced parameters
-        result = subprocess.run(
-            command,
-            stdin=subprocess.DEVNULL, # Prevent ffmpeg from waiting for stdin
-            capture_output=True,      # Capture stdout and stderr
-            text=True,                # Decode output as text
-            check=False               # Don't raise exception on non-zero exit code automatically
+    def _run_ffmpeg_command(cmd: List[str], *, label: str) -> subprocess.CompletedProcess:
+        if cancel_check is None:
+            return subprocess.run(
```

Ruff (0.14.13)

`subprocess` call: check for execution of untrusted input

(S603)